ci/travis/bazel_bashrc.inc.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
# shellcheck shell=bash
# ^ Avoid shebang line. (This script must be sourced, not executed.)
```
This is actually one of the few scripts that should probably never be executed, so you might want to remove the shebang line (so people get an error if they try to execute it) and even remove the execute bit (`chmod -x`). Instead you should be able to use [a directive](https://github.com/koalaman/shellcheck/wiki/SC2148) to indicate the language.